From: Jeff Mahoney <jeffm@suse.com>
Subject: ptp: PTP_1588_CLOCK_PCH depends on x86
Patch-mainline: no, rejected at https://patchwork.kernel.org/patch/2069071/

The PCH EG20T is only compatible with Intel Atom processors so it
should depend on x86.

Cc: Richard Cochran <richardcochran@gmail.com>
Signed-off-by: Jeff Mahoney <jeffm@suse.com>
---
 drivers/ptp/Kconfig |    1 +
 1 file changed, 1 insertion(+)

--- a/drivers/ptp/Kconfig
+++ b/drivers/ptp/Kconfig
@@ -72,6 +72,7 @@ config DP83640_PHY
 
 config PTP_1588_CLOCK_PCH
 	tristate "Intel PCH EG20T as PTP clock"
+	depends on X86
 	select PTP_1588_CLOCK
 	help
 	  This driver adds support for using the PCH EG20T as a PTP
